mozilla/foundation.mozilla.org
# updated petition snippets to be ordered newest to oldest

Closes #7615 

**Link to sample test page**: https://foundation-s-7615-petit-w9nftu.herokuapp.com/cms/snippets/wagtailpages/petition/?locale=en

**Steps to test**:

1. Please visit the testing link above, and log in to the CMS.
2. Create a new snippet with whatever name you like
3. Once saved, the new snippet should now appear at the top of the page vs at the end of the list.
4. If everything is working as expected, testing is complete!

## Checklist

**Changes in Models:**
